sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/Samples/Samples_ConversationAnalysisClient.cs

```diff
@@ -400,48 +528,79 @@ public async Task Example_AnalyzeConversations_ShortVersion_Async()
             {
 new
 {
+modality = "text",
 conversationItems = new object[]
 {
 new
 {
-text = "<text>",
 id = "<id>",
 participantId = "<participantId>",
+text = "<text>",
 }
 },
 id = "<id>",
 language = "<language>",
-modality = "text",
 }
             },
                 },
                 tasks = new object[]
             {
 new
 {
-kind = "ConversationalSummarizationTask",
+kind = "ConversationalPIITask",
 }
             },
             });
-            Operation<BinaryData> operation = await client.AnalyzeConversationsAsync(WaitUntil.Completed, content);
-            BinaryData responseData = operation.Value;
+            Operation operation = await client.AnalyzeConversationSubmitOperationAsync(WaitUntil.Completed, content);
+        }
 
-            JsonElement result = JsonDocument.Parse(responseData.ToStream()).RootElement;
-            Console.WriteLine(result.GetProperty("tasks").GetProperty("completed").ToString());
-            Console.WriteLine(result.GetProperty("tasks").GetProperty("failed").ToString());
-            Console.WriteLine(result.GetProperty("tasks").GetProperty("inProgress").ToString());
-            Console.WriteLine(result.GetProperty("tasks").GetProperty("total").ToString());
-            Console.WriteLine(result.GetProperty("createdDateTime").ToString());
-            Console.WriteLine(result.GetProperty("jobId").ToString());
-            Console.WriteLine(result.GetProperty("lastUpdatedDateTime").ToString());
-            Console.WriteLine(result.GetProperty("status").ToString());
+        [Test]
+        [Ignore("Only validating compilation of examples")]
+        public void Example_Conversations_AnalyzeConversationSubmitOperation_ShortVersion_Convenience()
+        {
+            Uri endpoint = new Uri("<https://my-service.azure.com>");
+            AzureKeyCredential credential = new AzureKeyCredential("<key>");
+            ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
+
+            MultiLanguageConversationInput conversationInput = new MultiLanguageConversationInput(new ConversationInput[]
+            {
+new TextConversation("<id>", "<language>", new TextConversationItem[]
+{
+new TextConversationItem("<id>", "<participantId>", "<text>")
+})
+            });
+            Operation operation = client.AnalyzeConversationSubmitOperation(WaitUntil.Completed, conversationInput, new AnalyzeConversationOperationAction[]
+            {
+new PiiOperationAction()
+            });
         }
 
         [Test]
         [Ignore("Only validating compilation of examples")]
-        public void Example_AnalyzeConversations_AllParameters()
+        public async Task Example_Conversations_AnalyzeConversationSubmitOperation_ShortVersion_Convenience_Async()
```

Comment:
What do you mean by convenience here? What is the test name pattern we're using?